tests/chainer_tests/links_tests/rnn_tests/test_link_n_step_rnn.py

```diff
@@ -413,4 +414,74 @@ def test_n_cells(self):
         assert self.rnn.n_cells == 1
 
 
+@testing.parameterize(
+    *testing.product(
+        {
+            'dtype': [numpy.float32, numpy.float64],
+            'initializer': ['random', None],
+            'activation_type': ['tanh', 'relu'],
+            'use_bi_direction': [True, False]
+        }
+    )
+)
+class TestInitialization(unittest.TestCase):
+    def setUp(self):
+        if self.initializer is None:
+            initializer = initializers.constant.Zero()
+
+        elif self.initializer == 'random':
+            initializer = initializers.GlorotUniform()
+
+        self.initialW = numpy.zeros((10, 10), dtype=self.dtype)
+        self.initial_bias = numpy.zeros((10, 1), dtype=self.dtype)
+
+        initializer(self.initialW)
+        initializer(self.initial_bias)
+
+        # FIXME (himkt) workaround
+        self.initial_bias = self.initial_bias.reshape(-1)
```

Comment:
Could you explain what issue is fixed by the workaround?

Reply:
I put this tricky workaround because NStepRNN accepts a `n`
dimensional ndarray and Initializer accepts `(n, 1)` ndarray.

If there is a good way to remove it, please let me know. :bow:

---

#### When `self.initial_bias = numpy.zeros(10, dtype=self.dtype)`

- Initializer:
`ValueError: shape must be of length >= 2. Actual shape: (10,)`

```python
chainer/initializers/uniform.py:115: in __call__
    fan_in, fan_out = initializer.get_fans(array.shape)
```

#### When `self.initial_bias = numpy.zeros((10, 1), dtype=self.dtype)`

- NStepRNN:
`ValueError: could not broadcast input array from shape (10,1) into shape (10)`

```python
chainer/links/rnn/n_step_rnn.py:85: in __init__
    b = variable.Parameter(bias_initializer, out_size)
chainer/variable.py:1805: in __init__
    data = initializers.generate_array(initializer, shape, xp)  # type: ignore # NOQA
chainer/initializers/__init__.py:74: in generate_array
    initializer(array)
chainer/initializers/constant.py:58: in __call__
    array, self.fill_value.astype(array.dtype, copy=False))
chainer/backend.py:63: in copyto
    numpy.copyto(dst, _cpu._to_cpu(src))
```

Reply:
It seems reasonable to me that `initial_bias=GlorotUniform()` is not allowed.

Reply:
e125f80 1544739
Replace GlorotUniform with Uniform in test and prohibit to use them for `initial_bias`.

Reply:
e125f80
Update to use `Uniform` instead of `GlorotUniform` for initializing a bias vector.